app/build.gradle

```diff
@@ -13,7 +13,7 @@ repositories {
 android {
     compileSdkVersion 29
     buildToolsVersion "29.0.3"
-    ndkVersion '21.0.6113669'
+    ndkVersion '21.1.6352462'
```

Comment:
The CI on this branch suddenly broke and I tracked it down to this: https://github.com/gradle/gradle/issues/12440

I thought this was the least bad workaround but happy to use another workaround.

Reply:
Ported to https://github.com/relaycorp/relaynet-courier-android/pull/63 to fix the latest dependabot PRs which are currently broken due to this.